docs/admin/authorization/rbac.md

```diff
@@ -78,42 +86,44 @@ metadata:
   name: read-pods
   namespace: default
 subjects:
-  - kind: User # May be "User", "Group" or "ServiceAccount"
-    name: jane
+- kind: User
+  name: jane
+  apiGroup: rbac.authorization.k8s.io
 roleRef:
   kind: Role
   name: pod-reader
   apiGroup: rbac.authorization.k8s.io
 ```
 
-`RoleBindings` may also refer to a `ClusterRole`. However, a `RoleBinding` that
-refers to a `ClusterRole` only applies in the `RoleBinding`'s namespace, not at
-the cluster level. This allows admins to define a set of common roles for the
-entire cluster, then reuse them in multiple namespaces.
+A `RoleBinding` may also reference a `ClusterRole` to grant the permissions to namespaced
+resources defined in the `ClusterRole` within the `RoleBinding`'s namespace.
+This allows administrators to define a set of common roles for the entire cluster,
+then reuse them within multiple namespaces.
 
 For instance, even though the following `RoleBinding` refers to a `ClusterRole`,
 "dave" (the subject) will only be able read secrets in the "development"
-namespace, the namespace of the `RoleBinding`.
+namespace (the namespace of the `RoleBinding`).
 
 ```yaml
 # This role binding allows "dave" to read secrets in the namespace "development"
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: read-secrets
-  namespace: development # This binding only applies in the "development" namespace
+  namespace: development # This only grants permissions within the "development" namespace
```

Comment:
- [ ] Since we're only granting permission for one resource (albeit with several verbs), consider using the singular "permission" here.  
  As written, I was expecting to find a reference to a role that allowed Dave to do multiple things.